pantos/common/restapi.py

```diff
@@ -19,6 +23,28 @@ def get(self):
         return None  # pragma: no cover
 
 
+class NodesHealthResource(flask_restful.Resource):
```

Comment:
This is the flask resource that can be easily added to any service, which will then return the json response with the health of the nodes.

This can now easily be added to any service with a one liner like here: https://github.com/pantos-io/servicenode/blob/main/pantos/servicenode/restapi.py#L310. Also, I can delete it from here and just copy paste this to every service, but IMO it's better to have it here

Reply:
I am not sure if this code should be here TBH. Common seems to become a Pantos component (SeerviceNode, ValidatorNode, etc) common and not a Pantos protocol common.

I believe the idea was that common was containing code which is reused across our services, but still be Pantos protocol generic.

What do you think?

Reply:
I disagree. I don't think that just defining a flask resource in common is making it a Pantos component, 
This is not yet added to a flask app or registered in some way. I agree with the way you defined the Pantos common, it is shared code across Pantos components and I see this piece of code exactly as that.

But I see where you are coming from, and I think that @juanmanuel-tirado was also sharing your point of view. So happy to move it there if that's the team agreement 👍

Reply:
I am aware that it has not been added to a component yet, but anyone who is using the common code also will get Flask related code. What if they use FastApi instead? Or completely something different? That's with what I mean being tied to our components :)

Our view is: Shared code between Pantos written components.

I hope that clarifies my statement :)

Reply:
IMHO, decoupling is preferable if it is meant to be reusable component :)

Reply:
Common has so far just been shared code between our Python projects. That has meant that if at least two projects need the same code, it goes into common. A REST resource is only actually callable in a project that uses common if it's added there as a resource (not automatically). So, I agree with @danut13 here. If common becomes too large at some point (I don't think that it currently is), we can always split it up in some reasonable way.

Reply:
> IMHO, decoupling is preferable if it is meant to be reusable component :)

I was under wrong assumption that we are only using `requests` in common. But we are already using flask in common.
So, changes are ok with me, until we decide to completely decouple it from flask in future.

Reply:
Then I will keep it as it as for now, unless we will have some other opinions